SMLHelper/Utility/ImageUtils.cs

```diff
@@ -48,20 +53,35 @@ public static Texture2D LoadTextureFromFile(string filePathToImage, TextureForma
         }
 
         /// <summary>
-        /// Creates a new <see cref="Atlas.Sprite" /> from an image file.
+        /// Creates a new <see cref="Sprite" /> from an image file.
         /// </summary>
         /// <param name="filePathToImage">The path to the image file.</param>
         /// <param name="format">
-        /// <para>The texture format. By default, this uses <see cref="TextureFormat.BC7" />.</para>
-        /// <para>https://docs.unity3d.com/ScriptReference/TextureFormat.BC7.html</para>
-        /// <para>Don't change this unless you really know what you're doing.</para>
+        /// The texture format. By default, this uses <see cref="TextureFormat.BC7" />.<br/>
+        /// https://docs.unity3d.com/ScriptReference/TextureFormat.BC7.html <br/>
+        /// Don't change this unless you really know what you're doing.
```

Comment:
Since when is `<br>` a valid tag in summaries?

Reply:
It's been a long time. Maybe they were always available.

`<br/>` is a line break  
`<para>` is a paragraph

Paragraphs always include extra space under them, so line breaks are preferred for keeping lines together in a block.

Reply:
I never knew they existed.